src/pages/Calculator.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react';
+import calculate from '../logic/calculate';
+// import '../styles/calculator.scss';
```

Comment:
Great job @Efiamotu-1 🤝

- [ ] [Optional] Kindly remove commented code from your project as this is regarded as a bad practice.

Reply:
thank you,